VAMobile/src/screens/PaymentsScreen/Copays/DisputeCopay/DisputeCopayScreen.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+
+import { useNavigationState } from '@react-navigation/native'
+import { StackScreenProps } from '@react-navigation/stack'
+
+import { FeatureLandingTemplate } from 'components'
+import { NAMESPACE } from 'constants/namespaces'
+import { PaymentsStackParamList } from 'screens/PaymentsScreen/PaymentsStackScreens'
+
+type DisputeCopayScreenProps = StackScreenProps<PaymentsStackParamList, 'DisputeCopay'>
+
+function DisputeCopayScreen({ navigation }: DisputeCopayScreenProps) {
+  const { t } = useTranslation(NAMESPACE.COMMON)
+  const prevScreen = useNavigationState((state) => state.routes[state.routes.length - 2]?.name)
```

Comment:
Can you please add a comment for why we're reaching for length - 2? Are we guaranteed for this not to error out?